src/docfx/config/Config.cs

```diff
@@ -177,20 +176,6 @@ private static void ValidateLocale(Config config)
             }
         }
 
-        private static void ValidateContributorConfig(ContributionConfig config, string configPath)
-        {
-            if (!string.IsNullOrEmpty(config.UserProfileCachePath)
-                && File.Exists(Path.Combine(configPath, config.UserProfileCachePath)))
-            {
-                throw Errors.UserProfileCacheNotFound(config.UserProfileCachePath).ToException();
-            }
-            if (!string.IsNullOrEmpty(config.GitCommitsTimePath)
-                && File.Exists(Path.Combine(configPath, config.GitCommitsTimePath)))
-            {
-                throw Errors.UserProfileCacheNotFound(config.GitCommitsTimePath).ToException();
-            }
-        }
```

Comment:
These check is still needed when using a local file. As it also supports url now, move to somewhere else?